### ISSUE TITLE
Input component property "maxlength" does not work

### ISSUE THREAD

**Issue by matthewchan15** [plotly/dash-core-components#505]
When using the `maxlength` property with the `input` component, nothing happens. The feature does not work. I've tested `maxlength` with both `string` and an `int` with no avail. I also tried setting `type=text` and that didn't work either.

This was tested with `dash-core-components==0.45.0`.

**Comment by matthewchan15** [comment]
Test code:
```
import dash
from dash.dependencies import Input, Output, State
import dash_core_components as dcc
import dash_html_components as html


app = dash.Dash(__name__)

app.layout = html.Div(
    [
        dcc.Input(
            id="test",
            type='text',
            maxlength='10'
        )   
    ],
)


if __name__ == '__main__':
    app.server.run(debug=True)
```

**Comment by celinehuang** [comment]
I have replicated the code in the latest version of Dash (`dash 1.1.1`) and this is no longer an issue @matthewchan15 could you close this issue since I don't have permission ?

**Comment by Marc-Andre-Rivet** [comment]
Closing as fixed as per #523 above.
